### ISSUE TITLE
Feedparser version not compatible with python 3.9

### ISSUE THREAD

**Issue by cl-parsons** [modoboa/modoboa#2326]
# Impacted versions

* OS Type: Alpine
* OS Version: 3.14
* Database Type: MySQL
* Modoboa: 1.17.0
* installer used: Yes
* Webserver: Nginx

# Steps to reproduce

modoboa-admin.py deploy instance --collectstatic --domain xxx --dburl xxxx

# Current behavior

```
  File "/opt/modoboa/venv/lib/python3.9/site-packages/feedparser.py", line 93, in <module>
    _base64decode = getattr(base64, 'decodebytes', base64.decodestring)
AttributeError: module 'base64' has no attribute 'decodestring'

['/opt/modoboa/venv/bin/python', 'manage.py', 'set_default_site', 'mail.freepoteries.fr'] failed, check your configuration
```

**Comment by cl-parsons** [comment]
This is fixed since feedparser v6.0.0. Here is the changelog for feedparser v6 : 

```
    Support Python 3.6, 3.7, 3.8 and 3.9
    Drop support for Python 2.4 through 2.7, and Python 3.0 through 3.5 (#169)
    Convert feedparser from a monolithic file to a package
    feedparser.parse(sanitize_html=bool) argument replaces the feedparser.SANITIZE_HTML global
    feedparser.parse(resolve_relative_uris=bool) replaces the feedparser.RESOLVE_RELATIVE_URIS global
    Unify the codebase so that 2to3 conversion is no longer required
    Remove references to iconv_codecs
    Update the Creative Commons namespace URI's
    Update the default User-Agent name and URL
    Support Middle European (Summer) Time timezones (#20)
    Pass data to lazy_chardet_encoding() (#50)
    Document that datetimes are returned in UTC (#51)
    Remove cjkpython references in the documentation (#57)
    Resolve ResourceWarnings thrown during unit tests (#170)
    Fix tox build failures (#213)
    Use base64.decodebytes() directly to support Python 3.9 (#201)
    Fix Python 3.8 urllib.parse.splittype() deprecation warning (#211)
    Support parsing colons in RFC822 timezones (#144)
    Add chardet as an optional tox environment dependency
    Fix the Big5 unit test that fails when chardet is installed (#184)
```

**Comment by cl-parsons** [comment]
This was already fixed in 2ab3bc3956f12d7311b8e7ad2a5ce285abdd7dba , so I think it should be released in modoboa 1.18.0!
